lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -60,6 +60,50 @@ function getNodesAndTimingByUrl(nodeTimings) {
   return urlMap;
 }
 
+/**
+ * Adjust the timing of a node and its dependencies to account for stack specific overrides.
+ * @param {Map<Node, LH.Gatherer.Simulation.NodeTiming>} adjustedNodeTimings
+ * @param {Node} node
+ * @param {LH.Artifacts.DetectedStack[]} Stacks
+ */
+function adjustNodeTimings(adjustedNodeTimings, node, Stacks) {
+  const nodeTiming = adjustedNodeTimings.get(node);
+  if (!nodeTiming) return;
+  const stackSpecificTiming = computeStackSpecificTiming(node, nodeTiming, Stacks);
+  const difference = nodeTiming.duration - stackSpecificTiming.duration;
+  if (!difference) return;
+
+  // Adjust timing of all dependent nodes
+  node.traverse(node => {
+    adjustedNodeTimings.delete(node);
```

Comment:
nit: shadowing can be confusing to those who are trying to understand it for the first time :)
```suggestion
  node.traverse(childNode => {
    adjustedNodeTimings.delete(childNode);
```